src/zstring.cpp

```diff
@@ -393,7 +393,6 @@ void FString::Remove(size_t index, size_t remlen)
 		}
 		else
 		{
-			remlen = Len() - remlen < remlen ? Len() - remlen : remlen;
```

Comment:
I think @coelckers has it right. I didn't know what to make of the line, but I never tried to delete out of string bounds, so it didn't matter removing it. I believe the console is currently the only code to use this method, so the bug wasn't noticed until now.